src/app/admin-dashboard/admin-dashboard.component.ts

```diff
@@ -30,4 +30,7 @@ export class AdminDashboardComponent implements OnInit {
     goToSubmitReviews(userId: string) {
         this.router.navigate(['/admin-dashboard/review/']);
     }
+    goToCreateUsers(userId: string){
+        this.router.navigate(['/admin-dashboard/create/', userId]);
```

Comment:
userId isn't needed here in the url.  You are calling the adminID from the `credentials()` function which takes local storage so it isn't needed.  It can actually be removed from this function as a parameters as well as the `goToSubmitReviews` above